backend/static/js/mapview.js

```diff
@@ -1,172 +1,172 @@
-            }
-        }
-
-        return markerArray
-    }
-}
-$.extend(mapViewPage.options, pageOptions)
-
-document.addEventListener("DOMContentLoaded", function domReady() {
-
-    mapViewPage.initializeMap()
-    mapViewPage.loadMapMarkers()
-    mapViewPage.registerEventHandlers(document, window)
-
-})
+
+
+mapViewPage = {
+    options: {
+        mapViewContainerId: '',
+        aURL : '',
+        bURL : '',
+        aListURL  : '',
+        bListURL   : '',
+        tileURL: '',
+        mapAttribution: '',
+        isStudent: true,
+        isHospital: true,
+        createPopupTextA  :  (countrycode, city, plz, count, url) => '',
+        createPopupTextB :  (countrycode, city, plz, count, url) => '',
+        createACountText: (count) => '',
+        createBCountText: (count) => '',
+        facilityIcon: new L.Icon.Default(),
+    },
+
+    mapObject: null,
+
+    createFacilityIcon: function createFacilityIcon(count) {
+        return L.divIcon({
+            className: 'leaflet-marker-icon marker-cluster marker-cluster-single leaflet-zoom-animated leaflet-interactive facilityMarker',
+            html: `<div><span>${count}</span></div>`,
+            iconSize: [40, 40],
+            popupAnchor: [-10,-10],
+        })
+    },
+
+    createSupporterIcon: function createSupporterIcon(count) {
+        return L.divIcon({
+            className: 'leaflet-marker-icon marker-cluster marker-cluster-single leaflet-zoom-animated leaflet-interactive supporterMarker',
+            html: `<div><span>${count}</span></div>`,
+            iconSize: [40, 40],
+        })
+    },
+
+    cssClassedIconCreateFunction: function cssClassedIconCreateFunction(cssClass) {
+        return (function (cluster) {
+            var childCount = cluster.getChildCount();
+            var cssClasses = ['marker-cluster']
+            var c = ' marker-cluster-'
+            if (childCount < 10) {
+                c += 'small'
+            } else if (childCount < 100) {
+                c += 'medium'
+            } else {
+                c += 'large'
+            }
+            cssClasses.push(c)
+            cssClasses.push(cssClass)
+            return new L.DivIcon({
+                 html: '<div><span>' + childCount + '</span></div>',
+                 className: cssClasses.join(' '),
+                 iconSize: new L.Point(40, 40)
+            })
+        })
+    },
+
+
+    initializeMap: function initializeMap() {
+        let mapOptions = {
+            center: [51.13, 10.018],
+            zoom: 6
+        }
+        let tileLayerURL = this.options.tileURL
+        let tileLayerOptions = {
+            attribution:  this.options.mapAttribution + '<a href="https://www.mapbox.com/map-feedback/" target="_blank">Improve this map</a> | Icons by <a href="https://www.flaticon.com/authors/freepik" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">flaticon.com</a>',
```

Comment:
@maltezacharias the attribution links that are contained in mapAttribution are rendered to text here (lower corner of the map). Any idea on how to fix this?

Reply:
see map.html

Reply:
Done